models/balances/optimism/erc20/balances_optimism_erc20_noncompliant.sql

```diff
@@ -0,0 +1,13 @@
+{{ config(
+        tags = ['dunesql'],
+        alias = alias('noncompliant'),
+        materialized ='table',
+        file_format = 'delta'
+        )
+}}
+
+SELECT  
+    DISTINCT token_address
+FROM 
+{{ ref('transfers_optimism_erc20_rolling_day') }}
+WHERE round(amount/power(10, 18), 6) < -0.001
```

Comment:
Good idea to check for compliance this way. We should monitor this table and see what ends up here so we can proactively identify if there's any faulty logic.

Also, give were using DuneSQL and have the INT256 and UINT256 types available, if everything runs as expected we shouldn't expect to see integer overflow or any weirdness happening. But still, better safe than sorry so let's monitor.